<launch>
    <node pkg="mag_viz" type="magnet_viz_node" name="magnet_viz" output="screen">
        <rosparam file="$(find mag_viz)/config/magnet_viz_node.yaml" command="load" />
    </node>
</launch>